packages/ndla-ui/src/Gloss/Gloss.tsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import styled from '@emotion/styled';
+import { colors, spacing, misc } from '@ndla/core';
+import { AccordionRoot, AccordionItem, AccordionHeader, AccordionContent } from '@ndla/accordion';
+import SpeechControl from '../AudioPlayer/SpeechControl';
+
+interface Example {
+  example: string;
+  language: string;
+}
+
+interface Alternatives {
+  traditionalChinese?: string;
+  pinyin?: string;
+  norwegianTranslation: string;
+}
+export interface Props {
+  title: {
+    title: string;
+    language: string;
+  };
+  glossData: {
+    glossType?: string;
+    alternatives: Alternatives;
+    originalLanguage: string;
+    examples?: Example[][];
+  };
+  audio: {
+    title: string;
+    src?: string;
+  };
+}
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  background-color: ${colors.background.lightBlue};
+  padding: ${spacing.nsmall} ${spacing.normal};
+  border: 1px solid ${colors.brand.lighter};
+  border-radius: ${misc.borderRadius};
+  margin-bottom: ${spacing.xsmall};
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+`;
+
+const GlossContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const GlossSpan = styled.span`
+  font-weight: 700;
+  padding-right: ${spacing.nsmall};
+`;
+
+const AlternativeSpan = styled.span`
+  padding-right: ${spacing.nsmall};
+`;
+
+const TypeSpan = styled.span`
+  padding-right: ${spacing.nsmall};
+  margin-bottom: 20px;
+  font-style: italic;
+`;
+
+const AudioExample = styled.div`
+  padding-bottom: ${spacing.normal};
+`;
+
+const ExampleHeader = styled(AccordionHeader)`
+  background-color: ${colors.background.lightBlue};
+`;
+
+const AccordianContainer = styled(AccordionContent)`
```

Comment:
Jeg er mer fan av å kalle komponenter som vi styler på nytt for `Styled<navn-på-komponent>` da det er lettere å gjenkjenne i html'en 😄  så `StyledAccordionHeader` og `StyledAccordionContent`?